src/java/org/apache/cassandra/auth/AuthCache.java

```diff
@@ -78,6 +94,8 @@ public static void shutdownAllAndWait(long timeout, TimeUnit unit) throws Interr
      * @param getUpdateIntervalDelegate Getter for update interval
      * @param setMaxEntriesDelegate Used to set max # entries in cache. See {@link com.github.benmanes.caffeine.cache.Policy.Eviction#setMaximum(long)}
      * @param getMaxEntriesDelegate Getter for max entries.
+     * @param setActiveUpdate Actively update the cache before expiry
```

Comment:
nit: Would this make more sense as something like "Set whether or not cache updates actively before expiry" or "Sets whether or not cache updates independent of expiry"?

Reply:
I'm struggling with the text here as well. Landed on "     * @param setActiveUpdate Method to process config to actively update the auth cache prior to configured cache expiration"